frontend/src/components/MissionOverview/UpcomingMissionView.tsx

```diff
@@ -151,7 +152,13 @@ export function UpcomingMissionView() {
                             onScheduleButtonPress={onScheduleButtonPress}
                             scheduleButtonDisabled={scheduleButtonDisabled}
                         ></ScheduleMissionDialog>
-                        <Button>Make new mission in Echo</Button>
+                        <Button
+                            onClick={() => {
+                                window.open(echoURL + savedAsset)
+                            }}
+                        >
+                            Create mission
```

Comment:
Why not specify Echo for create mission?

Reply:
As specified in https://github.com/orgs/equinor/projects/85
@aestene can you answer?